src/pages/Home/components/OrderCategories/OrderCategories.tsx

```diff
@@ -0,0 +1,46 @@
+import OrderCategoryDesserts from "../../../../utils/images/OrderCategoryDesert.jpg";
+import OrderCategoryDrinks from "../../../../utils/images/OrderCategoryDrinks.jpg";
+import OrderCategoryPizza from "../../../../utils/images/OrderCategoryPizza.jpg";
+import OrderCategory from "./OrderCategory";
+
+const OrderCategories = () => {
+  const ORDER_CATEGORY_DATA = [
+    {
+      image: OrderCategoryPizza,
+      label: "Pizza noastra",
+      className: "col-span-3 lg:row-span-1 lg:col-span-2 lg:row-span-2",
```

Comment:
Having the className declared here is not a good idea. In a real case-scenario this data comes from Back-End, which will never send you classNames. Please revise the implementation

Reply:
Revised the implementation. Now its using an if else